### ISSUE TITLE
3.2 crash on delete

### ISSUE THREAD

**Issue by fcore117** [nomacs/nomacs#60]
nomacs 3.2.0 crashes on delete, win 8.1 x64

**Comment by SaintDI** [comment]
Confirm.
ver. 3.2.0.174 [x64] Qt 5.5.1
Windows 10 (x64)

The error appears constantly. All reports indicate an error in the module Qt5Core.dll

Here, windows log:
```
Application Error
ID: 1000

Name of the failed application: nomacs.exe, Version: 3.2.0.174, timestamp: 0x5721f6e7
Name of the failed module: Qt5Core.dll, Version: 5.5.1.0, time stamp: 0x566eaaab
Exception code: 0xc0000005
Offset error: 0x0000000000004016
The ID of the failed process: 0x33d4
Start time of the failed application: 0x01d1ae1d58a41b5a
The path of the failed application: C: \ Program Files (x86) \ nomacs \ nomacs-x64 \ nomacs.exe
The path of the failed module: C: \ Program Files (x86) \ nomacs \ nomacs-x64 \ Qt5Core.dll
```
I hope that this information will be useful to you.

**Comment by diemmarkus** [comment]
Hi,
my problem is, that I cannot reproduce it even though I have the very same system settings. Could one of you be so kind and reproduce it with the [portable version](http://download.nomacs.org/portable/), and then send me the settings.nfo file? (I guess, I will be able to reproduce the crash then...)
thanks,
markus

**Comment by SaintDI** [comment]
Hmm... I tried to repeat the bug in the portable version. Everything works well! But deleting the same files using the installed version still lead the program to crash.

I recorded a short video about this. (And sorry for the last image, I don't imply anything)
[nomacs-crash.zip](https://github.com/nomacs/nomacs/files/265198/nomacs-crash.zip)
or [direct link](http://videobin.org/+ct5/ggw.html) (videobin)

P.S. Maybe the bug is somehow connected with the updating of the program? Because it has appeared only **after** the upgrade. I can completely reinstall nomacs (and tell, helped or not), or will try to help you find a bug in current state. Which options is preferable for the team?
Thx.

**Comment by diemmarkus** [comment]
Hi,

that suggests a user setting that is different from mine. If you find the time, it would be awesome if you could export your settings: 
- Open Windows Registry - _Win+R -> regedit_
- Go to nomacs settings - `HKEY_CURRENT_USER\SOFTWARE\nomacs\Image Lounge`
- Right click `Image Lounge` key -> Export
- Send me this file: <markus@nomacs.org> _(NOTE: it might contain personal data, so I would not share it publicly)_

thanks again

**Comment by diemmarkus** [comment]
awesome - I could reproduce it & fix it... thanks! (expect the fixed release of nomacs 3.2.1 mid of June)
